src/tile/tile.scss

```diff
@@ -97,10 +97,11 @@
     > .iui-picture {
       width: 100%;
       height: 100%;
-      transition: transform $iui-speed ease;
+      transition: width $iui-speed ease, height $iui-speed ease;
       position: absolute;
-      top: 0;
-      left: 0;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
```

Comment:
Um yeah I dont like this. Can we avoid `transform` everywhere?

Reply:
I agree because it also can randomly make stuff blury.